File: laboratory work 4/solutions/code/1.py
toContinueString='+';
while toContinueString!='-':
	if toContinueString=="+":
		x=0;
		sum=0;
		n=0;
		x=float(input("input x: "));
		n=int(input("input n: "));
		for i in range(1,n+1,1):
			value=x**i;
			sum += value;
			print(str(x)+"^"+str(i)+"= "+str(value));
		print("sum= "+str(sum));
	toContinueString=input("type '+' to repeat or '-' to exit: ");

File: laboratory work 4/solutions/code/1.cpp
#include <iostream>
#include <math.h>
using namespace std;


int main()
{
	string* con = new string("+");
	while (*con != "-")
	{
		if (*con == "+") {
			double* x = new double(0), * sum = new double(0);
			int* n = new int(0);
			cout << "input x: ";
			cin >> *x;
			cout << "input n: ";
			cin >> *n;
			for (int i = 1; i <= *n; i += 1) {
				double* ch = new double(pow(*x, i));
				*sum += *ch;
				cout << *x << "^" << i << "= " << *ch << endl;
				delete ch;
			}
			cout << "sum= " << *sum << endl;
			delete n;
			delete x;
			delete sum;
		}
		cout << "type '+' to repeat or '-' to exit: "; cin >> *con;
	}
	delete con;
	return 0;
}